portfolio/src/main/webapp/animations.js

```diff
@@ -0,0 +1,35 @@
+let isMenuOpen = false;
```

Comment:
FYI: It will become a problem as soon as you need to add second menu on the page. Normally it must be encapsulated inside "Menu" object. 

It's ok to keep as is for the codelab.

Reply:
+1